src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/AbstractEntityValidation.feature.disabled

```diff
@@ -1,18 +1,34 @@
 Feature: Client cannot POST an abstract entity
-  Ensure that clients cannot POST an abstract entity
+  Ensure that clients cannot interact with abstract entities
 
   @ignore
-  Scenario: Validation of an Abstract Entity
-    Clients cannot POST to the Education Organization abstract entity
+  Scenario: Ensure that clients cannot POST an abstract entity (Education Organization)
+    
     When sending a POST request to "/ed-fi/educationOrganization" with body
             """
             """
     Then it should respond with 404
 
   @ignore
-  Scenario: Ensure that clients cannot POST an abstract entity (Student Program Association)
-    Clients cannot POST to the Student Program Association abstract entity
-    When sending a POST request to "/ed-fi/studentProgramAssociations" with body
+  Scenario: Ensure that clients cannot POST an abstract entity (General Student Program Association)
+
+    When sending a POST request to "/ed-fi/generalStudentProgramAssociations" with body
+            """
+            """
+    Then the response code is 404
+
+  @ignore
+  Scenario: Ensure that clients cannot GET an abstract entity (Education Organization)
+    
+    When sending a GET request to "/ed-fi/educationOrganization" with body
+            """
+            """
```

Comment:
Since this is a GET request and will never have a body, please simplify the line above by removing "with body" and the triple quotes on the next two lines.

This establishes a new pattern that will also be useful in the future for DELETE requests.

Reply:
Done